### PR TITLE
MRG: watersheeeed

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

File: mne/bem.py [hunk 1]
```diff
@@ -1050,6 +1053,11 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
         .. versionadded:: 0.12
 
+    copy : bool
+        If True (default False), use copies instead of symlinks for surfaces
+        (if they do not already exist).
```
Comment (jona-sassenhagen) [inline mne/bem.py]: Usually we write "Defaults to False" right?

File: mne/bem.py [hunk 2]
```diff
@@ -1725,6 +1733,11 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
         within the subject reconstruction is used.
 
         .. versionadded:: 0.13.0
+    copy : bool
+        If True (default False), use copies instead of symlinks for surfaces
```
Comment (jona-sassenhagen) [inline mne/bem.py]: same

File: mne/commands/tests/test_commands.py [hunk 3]
```diff
@@ -85,24 +85,24 @@ def test_clean_eog_ecg():
...
+        fid.write('MEG 2443\n')
+    shutil.copyfile(raw_fname, use_fname)
+    with ArgvSetter(('-i', use_fname, '--bad=' + bad_fname,
+                     '--rej-eeg', '150')):
+        with pytest.warns(None):  # samples, sometimes
+            fun.run()
+    fnames = glob.glob(op.join(tempdir, '*proj.fif'))
+    assert len(fnames) == 1
+    fnames = glob.glob(op.join(tempdir, '*-eve.fif'))
+    assert len(fnames) == 1
```
Comment (jona-sassenhagen) [inline mne/commands/tests/test_commands.py]:
```
for files in ("*-eve.fif", "*proj.fif"):
    assert(len(glob.glob(op.join(tempdir, files))) == 1)
```
?

**Comment by jona-sassenhagen** [comment]
Only stylistic comments

File: mne/bem.py (hunk 1 above)
Comment (larsoner) [reply mne/bem.py]: It varies across the codebase

**Comment by larsoner** [comment]
Thanks @massich @jona-sassenhagen (tested locally after stylistic commit)

File: mne/bem.py (hunk 1 above)
Comment (massich) [reply mne/bem.py]: in bem it uses (default bla).

**Pull request merged**